src/search_algo.hpp

```diff
@@ -455,14 +514,14 @@ searchHalfExactImpl(LocalDataHolder<TGlobalHolder> & lH, TSeed && seed)
                     alph_t cur_letter = seqan3::assign_rank_to(r, alph_t{});
 
                     lH.cursor_tmp_buffer2.emplace_back(cursor, error_count + (cur_letter != seed_at_i));
-                    if (!lH.cursor_tmp_buffer2.back().first.extend_right(cur_letter))
+                    if (!extendRight(lH.cursor_tmp_buffer2.back().first, cur_letter))
```

Comment:
There is member `.extendRight()` above, but here you switc member for free function, why?